app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+<div class="container">
+<%= link_to "sample app", "#", id: "logo" %>
+    <nav>
+        <ul class="nav navbar-nav navbar-right">
+            <li><%= link_to "Home", "#" %></li>
+            <li><%= link_to "Help", "#" %></li>
+            <li><%= link_to "Log in", "#" %></li>
+        </ul>
+    </nav>
+</div>
+</header>
```

Comment:
1. thiếu end-line: là line trống cuối cùng của file, ko chứa j kể cả space
2. sai indent
3. Sử dụng I118n
-> em self check và fix cho toàn bộ các file html bên dưới